pkg/controller/volume/attach_detach_controller.go

```diff
@@ -309,10 +299,11 @@ func (adc *attachDetachController) processPodVolumes(
 
 		} else {
 			// Remove volume from desired state of world
-			uniqueVolumeName, err := attachableVolumePlugin.GetUniqueVolumeName(volumeSpec)
+			uniqueVolumeName, err := attachdetach.GetUniqueDeviceNameFromSpec(
+				attachableVolumePlugin, volumeSpec)
 			if err != nil {
 				glog.V(10).Infof(
-					"Failed to delete volume %q for pod %q/%q from desiredStateOfWorld. GetUniqueVolumeName failed with %v",
+					"Failed to delete volume %q for pod %q/%q from desiredStateOfWorld. GenerateUniqueDeviceName failed with %v",
```

Comment:
My understanding is that rule is for error strings, since they may be concatenated at a higher level before being printed. But log messages are basically the end of the line so it is ok to capitalize in a way that makes it user readable. 

GoLang guidence (https://github.com/golang/go/wiki/CodeReviewComments#error-strings) seems to align with this: `This does not apply to logging, which is implicitly line-oriented and not combined inside other messages.`